src/components/Group.vue

```diff
@@ -7,9 +7,8 @@
       type: 'Group',
       focusElement: true
     }"
-    @focus="focus"
-    @mousedown="focus"
-    :class="{ focused }"
+    tabindex="0"
```

Comment:
Why is the tabindex always 0?

Reply:
> tabindex="0" allows elements besides links and form elements to receive keyboard focus. It does not change the tab order, but places the element in the logical navigation flow, as if it were a link/button on the page.

https://webaim.org/techniques/keyboard/tabindex

Reply:
Thank you, was not aware of this difference!